### ISSUE TITLE
Hyperparameters and accuracy for AtrialFibrillation dataset.

### ISSUE THREAD

**Issue by letsGoBharat** [gzerveas/mvts_transformer#52]
Hello,

I am attempting to perform pre-training and fine-tuning on the AtrialFibrillation dataset, but I am unable to locate the hyperparameters and corresponding performance metrics in the relevant paper. Can you please provide me with this information, if available?

**Comment by letsGoBharat** [comment]
Is it possible to get some help with this? @gzerveas

**Comment by gzerveas** [comment]
Hi, this dataset was not included in the evaluation of the paper. If it is in the time series regression/classification archive, you can download it and put it in a directory as the README indicates, and the `class TSRegressionArchive(BaseData):` in the `data.py` should be able to handle it.
However, I can't really tell what would be good hyperparameters without having worked with this dataset. I suggest looking at the paper's table with default hyperparamters as a starting point, and experiment a bit (have a look at other datasets' hyperparameters to see which ones are the ones that most often affect performance). You may also consider using a sub-sample factor if the time series are very long or sampled with a high temporal resolution.
